Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -94,10 +100,25 @@ open class PassiveLocationManager: NSObject {
 
         lastRawLocation = locations.last
     }
+
+    /// Suspends the driving session.
+    ///
+    /// Use this method when you no longer need to receive updates of location status to preserve existing billing session.
+    public func pauseDriveSession() {
```

Comment:
Consider renaming to `stopTripSession()`: start/stop would be consistent with other PassiveLocationManager methods, and “trip” will be extensible to other modes of transportation that might be billed for.